spec/_attachments/ic.did

```diff
@@ -131,18 +131,36 @@ type bitcoin_get_balance_args = record {
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_result = satoshi;
+
 type bitcoin_get_balance_query_args = record {
     address : bitcoin_address;
     network : bitcoin_network;
     min_confirmations : opt nat32;
 };
 
+type bitcoin_get_balance_query_result = satoshi;
+
+type bitcoin_get_current_fee_percentiles_args = record {
+    network : bitcoin_network;
+};
+
+type bitcoin_get_current_fee_percentiles_result = vec millisatoshi_per_byte;
+
 type bitcoin_send_transaction_args = record {
     transaction : blob;
     network : bitcoin_network;
 };
 
-type millisatoshi_per_byte = nat64;
+type bitcoin_get_block_headers_args = record {
+	start_height : nat32;
+	end_height : opt nat32;
```

Comment:
nit: the rest of this file is using spaces rather than tabs.

Reply:
Done! Can you confirm that it has been fixed?

Reply:
Yes, looks good!